src/commands/build/IOSBuilder.js

```diff
@@ -583,7 +583,7 @@ See https://docs.expo.io/versions/latest/guides/building-standalone-apps.html`
       await this.runningAsCI(credsStarter, credsMetadata);
       this._areCredsMissing(credsStarter);
       await Credentials.updateCredentialsForPlatform('ios', credsStarter, credsMetadata);
-      log.warn(`Encrypted ${[...OBLIGATORY_CREDS_KEYS.keys()]} and saved to expo servers`);
+      log.warn(`Encrypted ${[...Object.keys(OBLIGATORY_CREDS_KEYS)]} and saved to expo servers`);
```

Comment:
Looks like the array spread (`[...]`) is no longer necessary.

Reply:
@fson  oh yes.i have update it, should i create a new pull request ?